src/Cartridge.cpp

```diff
@@ -58,7 +58,7 @@ Cartridge::Cartridge(const std::string& fileName) {
 		}
 	}
 	else {
-		printf("Error loading rom");
+		printf("Error loading rom\n");
```

Comment:
It'd probably make sense to convert all of these to work with the vita debug screen if building for vita